src/tribler/core/components/libtorrent/tests/conftest.py

```diff
@@ -15,10 +17,10 @@ def tdef():
 
 
 @pytest.fixture
-def mock_download_config(mocker, test_download):
-    return mocker.patch.object(test_download, 'config')
+def mock_download_config(test_download):
+    test_download.config = MagicMock()
 
 
 @pytest.fixture
-def mock_download_state(mocker, test_download):
-    return mocker.patch.object(test_download, 'get_state')
+def mock_download_state(test_download):
```

Comment:
It looks like the `mock_download_state` fixture is not used; I think we can delete it.